.github/CODEOWNERS

```diff
@@ -2,6 +2,7 @@
 
 services/queue @jhford
 services/web-server @taskcluster/frontend-reviewers
+taskcluster-web @taskcluster/frontend-reviewers
```

Comment:
You're going to hate me, sorry, but I kind of want to just call it `web/` or something else short and sweet (`frontend`? `ui`?).  We've eliminated `taskcluster-` from all of the other pathnames in this repo.  Hopefully once a name is selected this isn't a hard fix..

Reply:
Oh, let's call it `ui`, PLEASE!